ocaml/xapi/db_gc_util.ml

```diff
@@ -110,6 +110,13 @@ let gc_VIFs ~__context =
 let gc_PBDs ~__context =
   gc_connector ~__context Db.PBD.get_all Db.PBD.get_record (fun x->valid_ref __context x.pBD_host) (fun x->valid_ref __context x.pBD_SR) Db.PBD.destroy
 
+let gc_Cluster_hosts ~__context =
+  gc_connector ~__context Db.Cluster_host.get_all Db.Cluster_host.get_record
+    (fun x -> valid_ref __context x.cluster_host_host)
+    (* TODO: a PIF after https://github.com/xapi-project/xen-api/pull/3585 *)
```

Comment:
That PR has been merged now (but it's fine by me to do this later)

Reply:
I rebased the PR but forgot to remove this line, will fix.